Observations/general_obs.pro

```diff
@@ -102,7 +102,7 @@ IF N_Elements(complex_beam) EQ 0 THEN complex_beam=1
 IF N_Elements(n_pol) EQ 0 THEN n_pol=2
 IF N_Elements(precess) EQ 0 THEN precess=0 ;set to 1 ONLY for X16 PXX scans (i.e. Drift_X16.pro)
 IF N_Elements(gain_factor) EQ 0 THEN gain_factor=0.15
-IF N_Elements(max_sources) EQ 0 THEN max_sources=10000. ;maximum total number of source components to fit
+IF N_Elements(max_deconvolution_components) EQ 0 THEN max_deconvolution_components=10000. ;maximum total number of source components to fit

```

Comment:
`max_deconvolution_components`, `gain_factor`, `add_threshold`, and `independent_fit` should all be removed from `general_obs`, since they are set in `fhd_init`, and the default values here will override those and cause confusion. You will need to remove them everywhere in `general_obs`.

Reply:
This is probably a separate issue, but we should also look through the rest of `general_obs` and make sure we're not setting any other defaults that we don't want to.

Reply:
Done!